test/unit/deprecated.js

```diff
@@ -429,8 +429,8 @@ QUnit[ typeof Symbol === "function" && Symbol.toStringTag ? "test" : "skip" ](
 		obj[ Symbol.toStringTag ] = "Function";
 		fn[ Symbol.toStringTag ] = "Object";
 
-		assert.equal( jQuery.isFunction( obj ), false, "function-mimicking object" );
-		assert.equal( jQuery.isFunction( fn ), true, "object-mimicking function" );
+		assert.equal( jQuery.isFunction( obj ), true, "function-mimicking object" );
+		assert.equal( jQuery.isFunction( fn ), false, "object-mimicking function" );
```

Comment:
These changes are incorrect, function-mimicking objects are not callable and object-mimicking functions still are (and jQuery.isFunction is intended to test that characteristic).

Reply:
Well, but it didn't and a simple `typeof` told otherwise.

Reply:
Right, the failure was reported in #3600 and fixed in a16339b8933f115da3661f3d3f64854c3fa60bdc, and we still don't know why you want to revert that because you haven't provided a reduced test case demonstrating your unfulfilled expectations.